package.json

```diff
@@ -8,10 +8,12 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "webpack -d --watch & nodemon ./server/index.js",
-    "seed": "node ./database_reviews/fake-data.js"
+    "start": "webpack -d --watch & nodemon ./server/server.js",
+    "seed": "node ./database_reviews/fake-data.js",
```

Comment:
This script creates the import file, but how do you actually get it into mysql? I would update this script so it does both the creating of the file followed by the import, or include instructions in your readme about what to do once the seed data has been generated (ie, how to import).